src/Providers/DataGrid/Wijmo/Features/ToolTip.ts

```diff
@@ -32,8 +32,16 @@ namespace Providers.DataGrid.Wijmo.Feature {
 					this._setCellTooltip(_currTarget, ht.getColumn().binding, ht.row);
 				}
 			} else if (cellType === wijmo.grid.CellType.ColumnHeader) {
-				// If the Column Header is from a Group Column, we need to use a different approach that the regular header
-				if (_currTarget.classList.contains(Helper.Constants.CssClasses.ColumnGroup)) {
+				// If the Column Header is from a Group Column, we need to use a different approach than the regular header
+				// We can check if the current target is a ColumnGroup by checking its class and if it has columns or columnGroups
+				if (
+					_currTarget.classList.contains(Helper.Constants.CssClasses.ColumnGroup) &&
+					((ht.getColumn(true)._binding === undefined &&
+						(ht.getColumn(true) as wijmo.grid.ColumnGroup).columns !== undefined &&
+						(ht.getColumn(true) as wijmo.grid.ColumnGroup).columns.length > 0) ||
+						((ht.getColumn(true) as wijmo.grid.ColumnGroup).columnGroups !== undefined &&
+							(ht.getColumn(true) as wijmo.grid.ColumnGroup).columnGroups.length > 0))
```

Comment:
Maybe `ht.getColumn(true)` could be stored on a variable before the if. This way it would make the code cleaner and reduce the method calls to only one?

Reply:
Makes total sence @BenOsodrac ! Changes done